app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
So, where is the launcher icon for v25 and lower? 🤔

Reply:
@luongvo It's still referencing the default resource.

Reply:
@luongvo For other destiny, it will get resource from `mipmap-hdpi, mipmap-mdpi, mipmap-xhdpi, mipmap-xxhdpi, .etc`